metadata-ingestion/scripts/modeldocgen.py

```diff
@@ -490,17 +498,16 @@ def strip_types(field_path: str) -> str:
             )
         )
 
-        mce = MetadataChangeEventClass(
+        yield MetadataChangeEventClass(
             proposedSnapshot=d,
         )
-        events.append(mce)
-
-        mcp = MetadataChangeProposalWrapper(
+        yield MetadataChangeProposalWrapper(
             entityUrn=d.urn,
             aspect=SubTypesClass(typeNames=["entity"]),
         )
-        events.append(mcp)
-    return events
+        yield MetadataChangeProposalWrapper(
+            entityUrn=d.urn, aspect=d.browse_path_v2  # type: ignore
```

Comment:
not a huge fan of the type ignore

Reply:
Sooo how much do we care about this modeldocgen class? Originally I tried changing `DatasetSnapshot.pdl` to also allow BrowsePathV2, but ran into issues with that... may just be the fact I've never changed a pdl and don't know the steps. So this was my second option, so I didn't have to alter too much code -- just store it on the object and produce the MCP later. Alternatively I can change this to only produce MCPs rather than the MCE but honestly I have no clue what this class does.